src/main/scala/xiangshan/frontend/NewFtq.scala

```diff
@@ -539,6 +539,20 @@ class Ftq(implicit p: Parameters) extends XSModule with HasCircularQueuePtrHelpe
   val validEntries = distanceBetween(bpuPtr, commPtr)
   val canCommit = Wire(Bool())
 
+  val backendIpf = RegInit(false.B)
+  val backendIaf = RegInit(false.B)
+  val backendPcFaultIfuPtr = RegInit(FtqPtr(false.B, 0.U))
+  when (fromBackendRedirect.valid) {
+    backendIpf := fromBackendRedirect.bits.cfiUpdate.backendIPF
+    backendIaf := fromBackendRedirect.bits.cfiUpdate.backendIAF
+    when (fromBackendRedirect.bits.cfiUpdate.backendIPF || fromBackendRedirect.bits.cfiUpdate.backendIAF) {
+      backendPcFaultIfuPtr := ifuWbPtr_write
```

Comment:
Why ifuWbPtr_write here?
Although it is equivalent, using ifuWbPtr_write is confusing.

FYI, it is equivalent due to https://github.com/OpenXiangShan/XiangShan/blob/082b30d1b017bd096bce2e5fc66bac2ab749293a/src/main/scala/xiangshan/frontend/NewFtq.scala#L1122-L1134

Reply:
Would you recommend using ifuPtr_write? I think both options are acceptable and can be interpreted in different ways. I used ifuWbPtr_write because the elsewhen clause compares ifuWbPtr with backendPcFaultIfuPtr, where backendPcFaultIfuPtr is kind of like a register for ifuWbPtr.

Reply:
Ok, this is confusing to me without digging into other code.
I would recommend adding some comments.

Reply:
Fair enough. Comments added.